aten/src/ATen/native/mkldnn/xpu/detail/oneDNNContext.h

```diff
@@ -55,21 +55,37 @@ struct TORCH_XPU_API GpuStreamManager {
   static GpuStreamManager& Instance(); // Singleton
 
   dnnl::stream get_stream() {
-    c10::DeviceIndex device_index = c10::xpu::current_device();
-    TORCH_INTERNAL_ASSERT(device_index < c10::xpu::device_count());
-    return dnnl::sycl_interop::make_stream(
```

Comment:
Why did not upstream stream pool before? Confused...

Reply:
As discussed offline, we use an elegant method to implement a stream pool to ensure compatibility with future feature `external stream`.